packages/tests/ex-stars-basic.test.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+// extra tests to confirm Node’s behavior:
+// - multi star and mismatch should not be exported
+// - rhs-extra should map but fail with MODULE_NOT_FOUND
+// - literal/** is treated as literal and not exported
```

Comment:
these tests are great - but would it make sense instead to make fixtures to cover them? that way we'd match the existing test corpuse.